src/components/AboutPage/components/AboutDesription/AboutDescription.scss

```diff
@@ -1,14 +1,21 @@
 @import '../../../../shared/styles/variables';
 @import '../../../../shared/styles/mixins';
 
+.about-description {
+  padding: 1rem 2rem 1rem;
```

Comment:
you can write in this case: `padding: 1rem 2rem`

Reply:
Thank's, fixed